hibike/README.md

```diff
@@ -211,6 +211,8 @@ Device Type Enumeration:
 |         |                | 12           | enc_pos             | float      | yes   | yes    |
 |         |                | 13           | enc_vel             | float      | yes   | no     |
 |         |                | 14           | motor_current       | float      | yes   | no     |
+|  0x0B   | RFID           | 0            | id      | uint8_t | yes | no |
+|         |                | 1            | detect_tag | bool       | yes   | no |
```

Comment:
Tag detect is declared to be a bool in the README.  It's a uint8_t in the json and the code.

Please use only one type.  (I'd prefer bool)